src/CoreWCF.Primitives/src/CoreWCF/Security/Tokens/IssuedSecurityTokenParameters.cs

```diff
@@ -19,7 +19,7 @@ public class IssuedSecurityTokenParameters : SecurityTokenParameters
         private const string WsidPrefix = "wsid";
         private const string WsidNamespace = "http://schemas.xmlsoap.org/ws/2005/05/identity";
         private static readonly string s_wsidPPIClaim = string.Format(CultureInfo.InvariantCulture, "{0}/claims/privatepersonalidentifier", WsidNamespace);
-        internal const SecurityKeyType DefaultKeyType = SecurityKeyType.SymmetricKey;
+        public const SecurityKeyType DefaultKeyType = SecurityKeyType.SymmetricKey;
         internal const bool DefaultUseStrTransform = false;
```

Comment:
You can't make this public. Make a new const in the consuming class in the ConfigurationManager project with this value.